test/src/app/App.test.js

```diff
@@ -32,12 +32,11 @@ contract('App', function ([_, owner]) {
 
     it('initializes all app properties', async function () {
       this.app.version.should.eq(initialVersion);
-      this.app.directories.should.have.key(initialVersion);
+      this.app.directory.should.not.be.null
```

Comment:
This type of assertion doesn't work because if `this.app.directory` is ever `null`, it will fail with "cannot access property `should` of `null`".

I've used `should.exist(this.app.directory)` before, but you need to `const should = require('chai').should()` at the beginning of the file, or we could export it from the setup and do `const should = require('../../setup');`.

Additionally, this probably only checks for `null` but we don't want it to be `undefined` either.